pyproject.toml

```diff
@@ -34,16 +34,28 @@ dependencies = {file = ["requirements.txt"]}
 
 [project.optional-dependencies]
 test = [
+    "chardet>=2.3.0",
+    "databend_sqlalchemy>=0.5.2",
     "pytest",
     "mock",
     "minimock",
+    "psycopg2-binary",
     "pytest-cov",
     "pytest-runner",
+    "python-dateutil>=1.5.0",
+    "python-magic>=0.4.12",
     "sqlalchemy-greenplum",
     "sqlalchemy-hana",
     "starrocks",
     "snowflake-sqlalchemy"
 ]
+full = [
+    "chardet>=2.3.0",
+    "databend_sqlalchemy>=0.5.2",
```

Comment:
Can you please remove the version specifier for this? It means we can't use our git specific version in plaid/wrunner if we need to. It was wrong before